lib/matplotlib/tests/test_colors.py

```diff
@@ -1162,3 +1169,10 @@ def test_repr_html():
     html = cmap._repr_html_()
     assert len(html) > 0
     assert cmap.name in html
+
+
+def test_get_under_over_bad():
+    cmap = plt.get_cmap('viridis')
+    assert_array_equal(cmap.get_under(), cmap(0.0))
```

Comment:
I think this will be the ends of the colorbars and not the over/under specifically (they are the same for this case, but not in general) I think you just need to nudge these a bit to hit the over/under.

Reply:
This test handles the "default" cases (where the over/under are the ends of the colormap), and my changes to other tests above handle the "explicitly set" cases. I think the test _coverage_ is correct, but I agree it's a little unclear. I will re-write the tests to make that more clear.

Reply:
I read your comment again and I think I misunderstood the first time. I will fix this to use values outside `[0.0, 1.0]`.

Reply:
Excellent, looks great now!